app/mailers/notifications_mailer.rb

```diff
@@ -0,0 +1,47 @@
+class NotificationsMailer < ApplicationMailer
+  def cancel_booking booking, reason
+    @booking = booking
+    @reason = reason
+
+    app_name = t('app_name')
+    notification_subject = t('notifications_mailer.cancel_booking.subject')
+
+    mail(to: booking.user_email, subject: "[#{app_name}] #{notification_subject}")
+  end
+
+  def notify_upcoming_booking booking
+    @booking = booking
+
+    app_name = t('app_name')
+    notification_subject = t('notifications_mailer.notify_upcoming_booking.subject')
+
+    mail(to: booking.user_email, subject: "[#{app_name}] #{notification_subject}")
+  end
+
+  def notify_new_booking booking
+    users = booking.resource.resource_type_users
```

Comment:
@Aralnoth En esto que cambiamos el otro día parece que hay un problema. Al crear un evento me dice:

`undefined method `resource' for #<Booking:0x007f8fec13aeb0>
Did you mean?  resource_type`